backend/attendance-service/src/main/java/org/egov/repository/AttendanceStaffRepository.java

```diff
@@ -0,0 +1,33 @@
+package org.egov.repository;
+
+import lombok.extern.slf4j.Slf4j;
+import org.egov.repository.querybuilder.AttendanceStaffQueryBuilder;
+import org.egov.repository.rowmapper.AttendanceStaffRowMapper;
+import org.egov.web.models.AttendanceStaffSearchCriteria;
+import org.egov.web.models.StaffPermission;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+@Slf4j
+public class AttendanceStaffRepository {
```

Comment:
Rename it to just StaffRepository

Reply:
From all the repository classes word Attendance has been removed